watergun_assassin/authentication/tests.py

```diff
@@ -0,0 +1,54 @@
+from django.test import TestCase
+from django.urls import reverse
+from datetime import datetime
+
+
+class UserCreationTest(TestCase):
+    def test_valid_payload(self):
+        url = reverse('create_user')
+        username = 'admin'
+        password = 'kjlkjlkjlk123'
+        email = 'email@company.com'
+        data = {'username':username,'password':password,'email':email}
+        response = self.client.post(url,data) 
```

Comment:
Same here, space after comma. I won't point it out anymore in the review so that I'm not just adding comment noise. You can probably just do a search and replace.